modin/core/io/column_stores/parquet_dispatcher.py

```diff
@@ -306,9 +334,12 @@ def build_query_compiler(cls, path, columns, index_columns, **kwargs):
         new_query_compiler : BaseQueryCompiler
             Query compiler with imported data for further processing.
         """
+        storage_options = kwargs.pop("storage_options", {}) or {}
```

Comment:
Why do we pop storage_options of the kwargs?

Reply:
Looks like this line was moved on layer above from `call_deploy` layer to `build_query_compiler`, and `kwargs` which are used in `call_deploy` should not have `storage_options`

Reply:
Yes @prutskov you are correct.